tornado_http_auth.py

```diff
@@ -207,7 +207,7 @@ def authenticate_user(self, check_credentials_func, realm):
 
         auth_data = auth_header.split(None, 1)[-1]
         auth_data = base64.b64decode(auth_data).decode('ascii')
-        username, password = auth_data.split(':')
+        username, password = auth_data.split(':',maxsplit=1)
```

Comment:
The argument of split is changed and another pull is requested. 
Thanks for your package.